# Problem Set 1: Summary Statistics
### Jake Lee

---

## Import Libraries and Data
Data source: These data were used in the doctoral dissertation of Jeffrey Blend,
Department of Agricultural Economics, Michigan State University, 1998. The thesis was
supervised by Professor Eileen van Ravensway. Drs. Blend and van Ravensway kindly
provided the data. The data come from a telephone survey conducted by the Institute for
Public Policy and Social Research at MSU.

In [1]:
import pandas as pd
data = 'APPLE.DTA'
df = pd.read_stata(data)

## 1. Compare the summary statistics for demand for eco-labeled apples classified by gender. What can you infer from these summary statistics?

In [2]:
# restrict to male
male_df = df[df['male'] == 1]

# restrict to female
female_df = df[df['male'] == 0]

# new df for comparing by gender
comparison_df = pd.DataFrame({
    'male_eco_labeled': male_df['ecolbs'],
    'female_eco_labeled': female_df['ecolbs']
})

comparison_df.describe()

male_eco_labeled  female_eco_labeled
count        173.000000          487.000000
mean           1.466281            1.476728
std            2.596333            2.502953
min            0.000000            0.000000
25%            0.000000            0.000000
50%            1.000000            1.000000
75%            2.000000            2.000000
max           21.000000           42.000000

The comparison suggests that gender does not significantly affect demand for eco-labeled apples. It does reveal that the sample contains more female entries than male.

## 2. Obtain the correlation between demand for regular apples, demand for eco-labeled apples, price of regular apples, price of eco-labeled apples, and income. What can you infer from this correlation table?

In [3]:
# columns of interest
columns = ['reglbs', 'ecolbs', 'regprc', 'ecoprc']

# df of columns of interest
columns_df = df[columns]

# create correlation matrix (table)
columns_df.corr()

reglbs    ecolbs    regprc    ecoprc
reglbs  1.000000 -0.163024 -0.057797 -0.020411
ecolbs -0.163024  1.000000  0.008663 -0.098912
regprc -0.057797  0.008663  1.000000  0.830759
ecoprc -0.020411 -0.098912  0.830759  1.000000

We can infer that there is an expected strong positive correlation of 0.83 between regular prices and eco prices. This means that as regular prices increase, eco prices will also increase and vice-versa. 

However, there is also a weak negative correlation between demand for regular labeled and demand for eco-labeled. We can thus infer that different labels are weak substitutes.